### DISCUSSION TITLE
opening same gallery from 2 different links

### DISCUSSION THREAD

**Discussion by snote** [sachinchoolur/lightGallery#1263] [Q&A]
How can I achieve to open the same gallery from a button but also from a thumbnail.
ID have to be unique, so I can't use 2 ID's.

**Comment by sachinchoolur** [comment]
Yes, it is possible. You can use the lightGallery public method openGallery 
Docs - https://www.lightgalleryjs.com/docs/methods/#openGallery

**Reply by snote** [reply]
Mhhh... but with openGallery I am still bound on a unique id? I can open 2 galleries with one id but not vice verca...

I have one image from which I open a gallery plus a menue with 3 buttons. The 3 buttons switch between 3 galleries but Gallery-3 should be the same gallery as the one opened from the image.

https://codepen.io/snote/pen/eYepEve

**Reply by snote** [reply]
I still could not figure out how to do this...
I have an image on which I click and open a gallery with some buttons on the same layer. With these buttons I can switch between different galleries.
The first gallery I opened with the thumb, I can close with the buttons/switch to the next one but I can't get to open it again from the buttons as I would need to open a gallery with the same id which is not possible.

Sorry, a little difficult to describe...
Snote
